scripts/dashboard/publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+cd /app/scripts/dashboard
+
+echo "Saving files to S3"
+aws s3 cp ./chart.png s3://devopsacademy.com.au/progression-chart/chart.png
```

Comment:
Is versioning enabled?

If not, should you append the current date to the file name so we have a history?